modules/javafx.graphics/src/main/native-decora/SSEUtils.cc

```diff
@@ -183,3 +183,21 @@ void fsample(jfloat *map,
         }
     }
 }
+
+bool checkRange(JNIEnv *env,
```

Comment:
It would be helpful to add a comment indicating that this will return true if the range check fails and false if it is OK.

Reply:
@kevinrushforth thanks for your inputs. I thought for sometime about how to name this function like notInRange() or outOfRange() but since it is checking so many parameters i didn't change function name. I have added comment about what the function is doing as pointed by you.

Also @arapte mentioned a small nit offline about how we can keep src/dst arguments on same line instead of dividing them. I have updated that also.